calculator_gem.rb

```diff
@@ -0,0 +1,47 @@
+require "colorize"
+
+puts "Let us do some simple maths! What kind of math do you want to do?"
+operation = gets.chomp
+
+# Checks for acceptable operators.
+correct_op_answers = ["+", "add", "-", "subtract", "*", "multiply", "/", "divide", "**", "exponent", "^", "%", "modulus"]
```

Comment:
Consider changing this variable name to be a little clearer about what it represents. Something like `available_operations` would make it more obvious what role the variable will play in your code.